.circleci/config.yml

```diff
@@ -148,12 +152,23 @@ jobs:
       - setup_build_environment
       - snyk/scan:
           additional-arguments: --all-sub-projects --policy-path=.snyk
-  package-charts:
+  release-charts:
     executor: helm
+    working_directory: ~/repo
     steps:
       - checkout
+      - run:
+          name: Add release tag
+          command: |
+            git config --global user.email "${CIRCLE_USERNAME}@hypertrace.org"
```

Comment:
We generally tag before publishing - this appears to tag in between publish jars/images and helm charts.

Reply:
Yes, you are right. 

But at other places, we first tag, publish jars/images, and push the tags, and then upload helm charts.

So, do you suggest first tag, and than upload everything and publish tags after helm charts pushed.

For this PR, I had followed the workflow from hypertrace-ingester (https://github.com/hypertrace/hypertrace-ingester/blob/main/.circleci/config.yml) and pinot (https://github.com/hypertrace/pinot/blob/main/.circleci/config.yml) etc. which follow similar approach as this one.

Reply:
This is fine as is, thanks for jogging my memory - for context, when we were first implementing the release workflow we ran into issue with the multi-stage publish process. Because each job in the pipeline is filtered to a certain tag pattern (so it doesn't run on a standard merge), that tag needs to remain through each stage, and we swap in the new tag at the end.  There were a few other alternatives and their problems discussed here: https://github.com/hypertrace/pinot/pull/19#discussion_r481179127

Changing the local tags before publishing the chart is just a bit of a shortcut so we don't have to parse the tag to get the chart version, but they're not actually pushed until the last step.